MediatorPattern/ManagerMediator.cs

```diff
@@ -0,0 +1,22 @@
+namespace MediatorPattern
+{
+    class ManagerMediator : Mediator
+    {
+        public Colleague Customer { get; set; }
+        public Colleague Programmer { get; set; }
+        public Colleague Tester { get; set; }
+
+        public override void Send(string message, Colleague colleague)
+        {
+            if (colleague == Customer)
```

Comment:
What about using a Dictionary here? Extensible code which can add other types of colleagues in the future may be fun. I can also look into an implementation this week

Reply:
Great idea! Sure it will be better